pkg/tests/end_to_end_tests/jaeger_store_integration_tests/trace_compare.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaeger_integration_tests
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/kr/pretty"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
```

Comment:
Can you please add a comment why we needed to copy/modify this?

Reply:
@niksajakovljevic It is just a temporary solution and in the long run we can either import these as packages (after removing `_test` suffix) or download from jaeger repo.